### PR TITLE
#310 Align WorkerProductionPanels in the planet view and remove duplicate code

### PR DESCRIPTION
**Pull request by Replicant7e4**
This aligns the WorkerProductionPanels in the planet view and removes duplicate code used to update the UI components in the planet view. Originally the values were set when the UI components were initialized, and soon after that the "this.updatePanel()" was called, which updated the values again. Components are now initialized with "0", which should keep the same width for all components.

### CODE REVIEW

**Comment by Replicant7e4** [comment]
Aargh. It's not working as intended. It width is still uneven. Do not merge yet. Any comments to the duplicate code removal?

File: src/main/java/org/openRealmOfStars/game/States/PlanetView.java [hunk 1]
```diff
@@ -315,39 +298,36 @@ public PlanetView(final Planet planet, final boolean interactive,
...
-        Icons.ICON_TAX, ": " + planet.getTax(),
+        Icons.ICON_TAX, ": 0",
         "How many productions are converted to credits", listener);
     taxPanel.setAlignmentX(Component.LEFT_ALIGNMENT);
     taxPanel.setInteractive(interactive);
     panel.add(taxPanel);
 
-    metal = new IconLabel(null, Icons.getIconByName(Icons.ICON_METAL),
-        ": " + planet.getMetal());
+    metal = new IconLabel(null, Icons.getIconByName(Icons.ICON_METAL), ": 0");
```
Comment (tuomount) [inline src/main/java/org/openRealmOfStars/game/States/PlanetView.java]: Total metal might not fit into one zero. I had planet with 54 metal on surface and even the 4 wasn't fully visible. I think this should have four zeroes just to be safe.

**Comment by Replicant7e4** [comment]
Never mind. I had master branch in use where the fix was not included, so I thought for a minute, that it didn't work. But yeah, it's working as intended :)

**Comment by Replicant7e4** [comment]
> What width is not uneven. I tried this with value with two digits and it seemed to work with those. I noticed that if planet is already full of population death icon is not then shown, but this issue might have been earlier too.

Never mind. I had master branch in use where the fix was not included, so I thought for a minute, that it didn't work. But yeah, it's working as intended :)

**Comment by Replicant7e4** [comment]
> What width is not uneven. I tried this with value with two digits and it seemed to work with those. I noticed that if planet is already full of population death icon is not then shown, but this issue might have been earlier too.
> 
> I was wrong. If mining production is 20 final zero does not fit into label anymore. So these population and production values should be with two zeroes.

I think the + -button goes too much to right, if there is 00 used.

**Comment by tuomount** [comment]
[almostEnd.save.zip](https://github.com/tuomount/Open-Realms-of-Stars/files/3769020/almostEnd.save.zip)
Attaching save file where one planet has 16 people and quite good production you can see that not all information is fitting for that label space. Load the zip and unzip it to saves folder. Planet name is Omega Canrium III. Easiest find by clicking the Planets button.

**Comment by tuomount** [comment]
I think removing duplicate code is fine.

**Comment by Replicant7e4** [comment]
What do you think, if we would change the component this way?
Screenshot at 2019-10-24 21-57-47

**Comment by tuomount** [comment]
That could also work. But I just realized that single zero is enough for first values. Planet can only have 16 population. Problems were in second column where some of the values like metal production and research were based on actual value not on ": 0". Second column can have bigger values due production from the buildings, so there I think ": 00" would be better.

**Comment by Replicant7e4** [comment]
Ok changed the initial values to 00 on the IconLabels. Is that what you ment?

**Comment by tuomount** [comment]
Yes, exactly. That double zero is still missing from farmProd, mineProd and prodProd IconLabels.

iconlabels

**Comment by Replicant7e4** [comment]
> Yes, exactly. That double zero is still missing from farmProd, mineProd and prodProd IconLabels.
> 
> iconlabels

Yes, noticed the same. Working on it already.

**Comment by tuomount** [comment]
:+1: 
Merging

**Pull request merged**